src/cpp/expr2cpp.cpp

```diff
@@ -476,16 +476,16 @@ std::string expr2cppt::convert_code(
 
 std::string expr2cppt::convert_extractbit(const exprt &src)
 {
-  assert(src.operands().size()==2);
-  return convert(src.op0())+"["+convert(src.op1())+"]";
+  return convert(to_binary_expr(src).op0()) + "[" +
+         convert(to_binary_expr(src).op1()) + "]";
```

Comment:
Could we use `to_extractbit_expr` instead?

Reply:
done